stylesheet.css

```diff
@@ -0,0 +1,353 @@
+/********************************************************************************
+Welcome to the land of Style.
+Also, I'm learning CSS. So, there will be lots of notes to remind me of what
+things are for and such.
+
+Color scheme: I set everything to black because I needed place holders. Except for
+the body background and the nav links;
+*********************************************************************************/
+
+/********************************************************************************
+GENERAL
+*********************************************************************************/
+
```

Comment:
Suggest adding box-sizing: border-box to all for less fragile sizing.
* {
  box-sizing: border-box;
}